lib/class-wp-rest-widget-types-controller.php

```diff
@@ -398,6 +401,11 @@ public function get_item_schema() {
 					'default'     => '',
 					'context'     => array( 'view', 'edit', 'embed' ),
 				),
+				'is_multi'                    => array(
```

Comment:
I am not sure I love the name `is_multi`. Can we use the `is_singular` and invert the logic. `is_multi` doesn't really mean anything. `is_singular` is much more clear.

Reply:
I'm not fussed either way but the Customizer's widget model already uses `is_multi` and I wonder if we should maintain consistency there?

https://github.com/WordPress/wordpress-develop/blob/2382765afa36e10bf3c74420024ad4e85763a47c/src/wp-includes/class-wp-customize-widgets.php#L1005